app/models/mail_notifications/config_error.rb

```diff
@@ -0,0 +1,10 @@
+class ConfigError < MailNotification
```

Comment:
Since this is missing namespace, I'd prefer something more specific. This is too generic term that can be defined in our dependencies. How about `ConfigManagementError` or similar?